packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -26,3 +87,9 @@ export const onRouteUpdate = ({ location }, pluginOptions = {}) => {
 
   return null
 }
+
+export function onInitialClientRender(_, pluginOptions) {
+  if (pluginOptions.enableWebVitalsTracking) {
+    sendWebVitals(pluginOptions.dataLayerName)
```

Comment:
`dataLayerName` doesn't appear to be valid plugin option for GA (c&p?)
```suggestion
    sendWebVitals()
```